infrastructure/quick-deploy/gcp/all-in-one/variables.tf

```diff
@@ -90,6 +90,25 @@ variable "keda" {
   default = {}
 }
 
+# Chaos Mesh
+variable "chaos-mesh" {
+  description = "Chaos Mesh configuration"
+  type = object({
+    namespace                 = optional(string, "chaos-mesh")
+    chaosmesh_image_name      = optional(string, "ghcr.io/chaos-mesh/chaos-mesh"),
+    chaosmesh_image_tag       = optional(string, "v2.6.3"),
```

Comment:
Please use the `versions.tfvars.json` to provide the default tag.
For GCP, you would need to add the image to the GAR: https://github.com/aneoconsulting/ArmoniK/blob/main/infrastructure/quick-deploy/gcp/all-in-one/gar.tf